build.gradle.kts

```diff
@@ -1,15 +1,25 @@
+import org.hypertrace.gradle.publishing.HypertracePublishExtension
+import org.hypertrace.gradle.publishing.License
+
 plugins {
   id("org.hypertrace.repository-plugin") version "0.1.2"
   id("org.hypertrace.ci-utils-plugin") version "0.1.1"
   id("org.hypertrace.jacoco-report-plugin") version "0.1.0" apply false
   id("org.hypertrace.docker-java-application-plugin") version "0.8.1" apply false
   id("org.hypertrace.docker-publish-plugin") version "0.8.1" apply false
+  id("org.hypertrace.publish-plugin") version "0.3.3" apply false
```

Comment:
Why do we need this?

Reply:
This is the hypertrace gradle publish plugin https://github.com/hypertrace/hypertrace-gradle-publish-plugin

Reply:
Sorry, my question is why do we want to publish config-bootstrapper as lib artifacts? In the e2e test, can that also be run within the test container?